src/AuthModule/Components/Register/Register.tsx

```diff
@@ -1,7 +1,432 @@
+                fontWeight: 600,
+              }}
+            >
+              Stay
+              <Typography
+                component="span"
+                variant="h5"
+                sx={{ color: "#152c5b" }}
+              >
+                cation.
+              </Typography>
+            </Link>
+          </Box>
+
+          <Box
+            // bgcolor={'red'}
+            sx={{
+              my: 8,
+              mx: 4,
+            }}
+          >
+            <Container maxWidth="sm">
+              <Box
+                sx={{
+                  my: 3,
+                }}
+              >
+                <Typography
+                  fontWeight={600}
+                  variant="h5"
+                  sx={{
+                    my: 4,
+                  }}
+                >
+                  Sign up
+                </Typography>
+                <Typography variant="body1">
+                  If you already have an account register <br /> You can{" "}
+                  <Link
+                    to={"/Authentication"}
+                    style={{
+                      color: "#eb5148",
+                      textDecoration: "none",
+                      fontWeight: 600,
+                      marginLeft: 5,
+                    }}
+                  >
+                    Login here !
+                  </Link>
+                </Typography>
+              </Box>
+              <Box
+                component="form"
+                noValidate
+                sx={{ mt: 1 }}
+                onSubmit={handleSubmit(handleRegister)}
+              >
+                <Box
+                sx={{ display: "flex", flexDirection:'column', alignItems: "center" }}
+                >
+                  <Box sx={{width:'100px',height:'100px'}}><img src={userImage} alt="" style={{width:'100%'}} /></Box>
+                <Button
+                
+                  component="label"
+                  role={undefined}
+                  variant="outlined"
+                  tabIndex={-1}
+                  // startIcon={<CloudUploadIcon />}
+                  sx={{mb:2}}
+                >
+                  Upload image
+                  <VisuallyHiddenInput type="file" 
+                  {...register("profileImage", { required: "Image is required" })}
+                  onChange={(e)=>setUserImage(URL.createObjectURL(e?.target?.files[0]))}
+                  />
+                </Button>
+                {errors.profileImage && (
+                    <Typography variant="body2" sx={{ color: "red" }}>
+                      {errors?.profileImage?.message}
+                    </Typography>
+                  )}
+                </Box>
+
+                <Box>
+                  <Typography sx={{ mt: 1 }}>User Name</Typography>
+                  <TextField
+                    // sx={{bgcolor:'#F5F6F8'}}
+                    placeholder="Please type here ..."
+                    margin="normal"
+                    required
```

Comment:
We can remove this attribute to be able to access the validations and errors from react-hook-form when the textfield is empty.